src/components/DonationForm/DonationForm.jsx

```diff
@@ -113,16 +131,50 @@ function DonationForm() {
         </div>
 
         <div className={styles['field-section']}>
-          <h1 className={styles.title}>Items to be Donated</h1>
-          <Input />
+          <h1 className={styles.title}>Furniture Submissions</h1>
+          {fields.map((furniture, index) => {
+            return (
+              // eslint-disable-next-line react/no-array-index-key
+              <div key={index}>
+                <h1 className={styles.title}> Furniture {index + 1} </h1>
+                <div className={styles['field-spacing']}>
+                  <FormLabel>Items to be Donated</FormLabel>
+                  <Select
+                    defaultChecked="Dressers"
+                    {...register(`furnitureField.${index}.itemName`)}
+                  >
+                    {furnitureOptions.map((furnitureItem, i) => {
+                      // eslint-disable-next-line react/no-array-index-key
+                      return <option key={i}>{furnitureItem}</option>;
+                    })}
+                  </Select>
+                </div>
+                <div className={styles['field-spacing']}>
+                  <FormLabel>Furniture Image Link</FormLabel>
+                  <Input defaultValue="" {...register(`furnitureField.${index}.imageLink`)} />
+                </div>
+                <div className={styles['field-spacing']}>
+                  <FormLabel>Description</FormLabel>
+                  <Input defaultValue="" {...register(`furnitureField.${index}.description`)} />
+                </div>
+                <Button onClick={() => remove(index)}>Delete</Button>
+              </div>
+            );
+          })}
```

Comment:
Extract these back to the `FurnitureFields` component, and pass in any dependencies through props. If you have any trouble extracting it while keeping the same functionality (as in keeping it synced with the rest of the form data) let us know.